app/assets/stylesheets/lp.css

```
@@ -36,6 +36,7 @@
 @import "./lp/blocks/lp/_lp-price.css";
 @import "./lp/blocks/lp/_lp-actions.css";
 @import "./lp/blocks/lp/_lp-company-logos.css";
+@import "./lp/blocks/lp/_lp-company-names.css";
```

Stylelint (17.12.0)

[error] Invalid position for `@import` rule (no-invalid-position-at-import-rule)

(no-invalid-position-at-import-rule)